content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md

```diff
@@ -128,15 +128,10 @@ Make sure that in the current branch:
   ```bash
   php bin/console prestashop:licenses:update
   ```
-  
-* All controllers are secured by annotations, and legacy link are provided for Symfony routes:
-  
-  ```bash
-  php bin/console prestashop:linter:security-annotation
-  php bin/console prestashop:linter:legacy-link
-  ```
```

Comment:
This part mus be kept, but mentioning it is only necessary until 8.2

Reply:
php bin/console prestashop:linter:security-annotation
  php bin/console prestashop:linter:legacy-link